ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -393,14 +393,14 @@ classes:
 
             storageDevice:
                 label: Storage Dev
-                details_display: true
+                details_display: false
 
             getStorageDevice:
                 label: Storage Device
                 api_only: true
                 api_backendtype: method
                 grid_display: true
-                details_display: false
+                details_display: true
```

Comment:
Does `getStorageDevice` method return correct values in all cases? It looks like method can return `self.type` sometimes.